documentation/2.0/content/concepts/archive.md

```diff
@@ -69,7 +69,7 @@ The root directory under which JARs/directories used for server classpaths are s
 The root directory under which empty directories must exist for Coherence persistent stores.
 
 #### `wlsdeploy/custom`
-This is the root directory where your custom files can be stored in and extracted from the archive. These files are not collected by the Discover tool. Every file resource under this directory is extracted.
+This is the root directory where your custom files can be stored in and extracted from the archive. These files are not collected by the Discover Domain Tool. Every file resource under this directory is extracted.
```

Comment:
"can be stored in and" --> "can be stored and"
"Every file resource under this directory is extracted" --> "Every file resource under this directory is extracted during `createDomain`". and update?

Reply:
@ddsharpe Changed this to, "Every file resource under this directory is extracted during `createDomain` and `updateDomain`. Do you want me to await verification (from whom?) before committing this change?